components/Navigation.js

```diff
@@ -36,17 +59,60 @@ class Navigation extends React.Component {
         break;
       }
     }
-
+    this.graphMenuClose();
   }
+
   render () {
+    var view = this.props.mainView;
     return (
       <div>
-      <button id='navBtnGraph' onClick={(evt) => this.navButtonClick(evt)} >Graph</button>
-      <button id='navBtnGraphRelative' onClick={(evt) => this.navButtonClick(evt)} >Relative</button>
-      <button id='navBtnMovieDetails' onClick={(evt) => this.navButtonClick(evt)} >MovieDetails</button>
-      <button id='navBtnMap' onClick={(evt) => this.navButtonClick(evt)} >Map</button>
-      <button id='navBtnTwitter' onClick={(evt) => this.navButtonClick(evt)} >Twitter</button>
-    </div>
+        <RaisedButton
+          id='navBtnGraph'
+          // onClick={(evt) => this.navButtonClick('navBtnGraph')}
+          onClick={this.graphButtonClick}
+          label="Graph"
+          primary={view === 'graphAbsolute' || view === 'graphRelative' ? true : false}
+        ></RaisedButton>
+        <Popover
+            open={this.state.isGraphMenuOpen}
+            anchorEl={this.state.anchorEl}
+            anchorOrigin={{horizontal: 'left', vertical: 'bottom'}}
+            targetOrigin={{horizontal: 'left', vertical: 'top'}}
+            onRequestClose={this.graphMenuClose}
+            autoCloseWhenOffScreen
+            canAutoPosition={true}
+            animated={true}
+          >
+            <Menu>
+              <MenuItem primaryText="Sort by Date" onClick={(evt) => this.navButtonClick('navBtnGraph')} />
+              <MenuItem primaryText="Sort by Relative to Release Date" onClick={(evt) => this.navButtonClick('navBtnGraphRelative')} />
+            </Menu>
+          </Popover>
+        {/* <RaisedButton
```

Comment:
What is this comment stuff doing in the curly braces? Not a complaint, just a question

Reply:
its how you comment inside the render for jsx